src/fdtdx_studio/project/project.py

```
@@ -119,21 +131,23 @@ async def localproject_load(self):
         )
     # checks if teh localstorage was empty
     if (len(importProject)>0):
         # tries to load the scene via the importer
       try:
-        self.controller.open_Project(Project(controller=self.controller, File= importProject))
+        loaded_project = await Project.create_from_file(self.controller, File=importProject) 
+        self.controller.open_Project(loaded_project)
         self.controller.ui_update()
         return True
         # if the import fails prints error messages and loads empty scene
       except:
```

Ruff (0.15.10)

[error] Do not use bare `except`

(E722)